config-server/src/main/resources/bootstrap.yml

```diff
@@ -0,0 +1,18 @@
+spring:
+  application:
+    name: config-server
+  profiles: default
+  cloud:
+    config:
+      server:
+        git:
+          uri: https://placeholder
+          search-paths: placeholder
+
+server:
+  port: 8888
+
+eureka:
+  client:
+    serviceUrl:
+      defaultZone: http://localhost:8761/eureka/
```

Comment:
This wont  work
localhost is the configserver and not eureka

Reply:
Changed to placeholder